lib/publiccloud/gce.pm

```diff
@@ -260,6 +260,27 @@ sub stop_instance {
     die("Failed to stop instance $instance_id") unless ($attempts > 0);
 }
 
+sub suspend_instance {
+    my ($self, $instance) = @_;
+    my $instance_id = $instance->instance_id();
+
+    die("Cannot suspend instance which is not running.") if ($self->get_state_from_instance($instance) ne 'RUNNING');
```

Comment:
```suggestion
    die("Cannot suspend instance which is not running.") if (lc($self->get_state_from_instance($instance)) ne 'running');
```

Nitpick: Let's make this comparison case insensitive (also on the following comparisons here)